src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+server.port=5000
+spring.data.mongodb.uri=mongodb+srv://db_user:[password]j@cluster0-nomfh.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
You will not be connecting to the "`test`" database, the correct database name is "`InternHome`".

Reply:
Done